lib/cc/engine/analyzers/go/main.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "flay"
+require "json"
+require "cc/engine/analyzers/reporter"
+require "cc/engine/analyzers/analyzer_base"
+
+module CC
+  module Engine
+    module Analyzers
+      module Go
+        class Main < CC::Engine::Analyzers::Base
+          LANGUAGE = "go".freeze
```

Comment:
Most of these `#freeze` calls are redundant because of the magic `frozen_string_literal` comment at the top of the file.